api/proxy/Makefile

```diff
@@ -15,7 +15,7 @@ clean:
 
 docker-build:
 	# we only use this to build the docker image locally, so we give it the dev tag as a reminder
-	cd ../.. && docker buildx bake proxy --load
+	cd ../.. && SEMVER=$(GIT_TAG) GIT_SHORT_SHA=$(GIT_COMMIT) GITDATE=$(BUILD_TIME) docker buildx bake proxy --load
```

Comment:
where does `GIT_SHORT_SHA` come from? looking at other services in the mono Dockerfile only showcases build context for `(SEMVER, GITCOMMIT, GITDATE)`